moveit_ros/planning/collision_plugin_loader/src/collision_plugin_loader.cpp

```diff
@@ -38,7 +38,7 @@
 
 namespace collision_detection
 {
-static const rclcpp::Logger LOGGER = rclcpp::get_logger("collision_plugin_loader");
+static const rclcpp::Logger LOGGER = rclcpp::get_logger("moveit.ros_planning.collision_plugin_loader");
```

Comment:
```suggestion
static const rclcpp::Logger LOGGER = rclcpp::get_logger("moveit.ros.planning.collision_plugin_loader");
```
The same for all moveit_ros -> moveit.ros